### ISSUE TITLE
F# interactive logo template is not populated with versions

### ISSUE THREAD

**Issue by nightroman** [fsharp/FSharp.Compiler.Service#941]
## F# interactive logo template is not populated with versions

**Repro steps**

(1) Build an `fsi.exe` like tool using one of

- [FsiExe sample source](https://github.com/fsharp/FSharp.Compiler.Service/tree/master/fcs/samples/FsiExe)
- ["Embedding F# Interactive" instructions](http://fsharp.github.io/FSharp.Compiler.Service/interactive.html)

(2) Invoke it to start an interactive session

As a result, it prints its logo with missed versions:

```
Microsoft (R) F# Interactive version  for F#
Copyright (c) Microsoft Corporation. All Rights Reserved.

For help type #help;;
```

**Expected behavior**

The logo should contain versions of interactive and F#, just like `dotnet fsi` and Visual Studio `fsi.exe` do:

```
> dotnet fsi

Microsoft (R) F# Interactive version 10.9.1.0 for F# 4.7
...


> ...\fsi.exe

Microsoft (R) F# Interactive version 10.9.1.0 for F# 4.7
...
```

**Actual behavior**

The F# interactive built using FSharp.Compiler.Service logo misses expected versions.

**Known workarounds**

None

**Related information**

Presumably, here is the use of template in the source:

https://github.com/fsharp/FSharp.Compiler.Service/blob/e0fbb597584111431b841389b3ca8f700921c227/src/utils/CompilerLocationUtils.fs#L17-L26

**Comment by nightroman** [comment]
May I get to know if this is going to be addressed? If it is not, it's fine. Then I will work around this issue somehow. But I need to know.

P.S. I do not want to present this kind of logo to my app users. Because the app "does not look quite right" from its very start. The app is `fsx.exe` of the [FSharpFar](https://github.com/nightroman/FarNet/blob/master/FSharpFar/README.md) package. It is effectively `fsi.exe` with some extensions related to FSharpFar.

**Comment by nightroman** [comment]
I can see it fixed in the just released 36.0.2.
Closing, thank you all!

**Comment by baronfel** [comment]
Thank you for testing so quickly!
